src/module/server-instance/view/server-instance.html

```diff
@@ -17,9 +17,9 @@
                 uib-dropdown-toggle><span translate>ACTIONS</span><span class="caret"></span>
         </button>
         <ul ng-if="ctrl.isDomainMode()" class="dropdown-menu" role="menu" aria-labelledby="dropdownMenu">
-          <li role="presentation" ng-class="ctrl.serverInstance.isRunning()?'disabled':''"><a id-generator="dropdown_menu.start" role="menuitem" ng-click="ctrl.createStartModal()" translate>START</a></li>
-          <li role="presentation" ng-class="!ctrl.serverInstance.isRunning()?'disabled':''"><a id-generator="dropdown_menu.stop" role="menuitem" ng-click="ctrl.createStopModal()" translate>STOP</a></li>
-          <li role="presentation" ng-class="ctrl.serverInstance.isRunning()?'disabled':''"><a id-generator="dropdown_menu.remove" role="menuitem" ng-click="ctrl.createRemoveModal()" translate>REMOVE</a></li>
+          <li role="presentation" ng-class="ctrl.canStart(ctrl.serverInstance)?'':'disabled'"><a id-generator="dropdown_menu.start" role="menuitem" ng-click="ctrl.createStartModal(ctrl.serverInstance)" translate>START_RELOAD_RESTART</a></li>
```

Comment:
Yeah maybe being completely dyanamic would be strange. Three separate menu items that are always present and then only enabled when appropriate could be less confusing? That way the user knows of those options, but also knows that only the enabled menu entries are appropriate for the current state